packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -399,8 +399,8 @@ export default abstract class SqlIntegration
             SELECT
               date,
               COUNT(*) as count,
-              ${this.avg("value")} as mean,
-              ${this.stddev("value")} as stddev
+              ${this.avg("coalesce(value,0)")} as mean,
```

Comment:
Because _byDateOverall and _overall are both downstream of `_metric` and we only do inner joins between them, wouldn't just changing L360 to be `coalesce(${aggregate, 0)` be a one line change that removes the need for these lines?

Seems like either approach should work, just wondering.